diff --git a/node_modules/@remix-run/netlify/server.js b/node_modules/@remix-run/netlify/server.js
index b8a5db2..87e9fae 100644
--- a/node_modules/@remix-run/netlify/server.js
+++ b/node_modules/@remix-run/netlify/server.js
@@ -27,15 +27,9 @@ function createRequestHandler({
   };
 }
 function createRemixRequest(event) {
-  let url;
-
-  if (process.env.NODE_ENV !== "development") {
-    url = new URL(event.rawUrl);
-  } else {
-    let origin = event.headers.host;
-    let rawPath = getRawPath(event);
-    url = new URL(rawPath, `http://${origin}`);
-  }
+  let origin = event.headers.host;
+  let rawPath = getRawPath(event);
+  let url = new URL(rawPath, `http://${origin}`);
 
   let init = {
     method: event.httpMethod,
